src/symbolize/gimli/coff.rs

```diff
@@ -54,15 +54,15 @@ impl<'a> Object<'a> {
         let mut i = 0;
         let len = symtab.len();
         while i < len {
-            let sym = symtab.symbol(i).ok()?;
+            let sym = symtab.symbol(SymbolIndex(i)).ok()?;
             i += 1 + sym.number_of_aux_symbols as usize;
             let section_number = sym.section_number.get(LE);
             if sym.derived_type() != object::pe::IMAGE_SYM_DTYPE_FUNCTION || section_number == 0 {
                 continue;
             }
             let addr = usize::try_from(sym.value.get(LE)).ok()?;
             let section = sections
-                .section(usize::try_from(section_number).ok()?)
+                .section(SectionIndex(usize::try_from(section_number).ok()?))
```

Comment:
This could be done using `sym.section()` instead (but then the section 0 check will need to be in its own `if let`).

Reply:
Done!